### DISCUSSION TITLE
怎么对已翻译完成的中文内容进行审阅

### DISCUSSION THREAD

**Discussion by 0xPabloLI** [PlanckerLabs/remix-cc#45] [General]
对包括不同社区已经单独或协同完成的中文内容进行系统化审阅，该如何更有效的分配、协同和标记审阅工作？

或者可以后期宣传Remix China相关工作的时候，请读者来github或者其他指定的地方提交发现的问题或者疑问，一并纠正。

**Comment by drafish** [comment]
任务分配就直接用这个仓库的 issue 来做好了。我们建 issue ，或者有感性兴趣的小伙伴自己建 issue ，描述下这个任务要审阅哪些模块。完成任务的小伙伴在 issue 里面评论并 @下我或者别的负责人，然后相关负责人确认任务真的已经完成后就可以把 issue 关了。
至于你说的协同和标记，是不是指怎么协作，怎么标识某一段文字翻译已经审阅过了。
我是准备用 crowdin 来协作翻译的，以太坊官网用的就是这个。Remix 团队也把 Remix IDE 的使用文档放在 crowdin 上。任何人都可以在上面协作翻译，而且可以对别人已翻译的内容进行投票，你觉得翻译的对就投赞成票，不对就投反对票。大家也可以对某一段文字进行评论。你可以点下面的链接进去看下
https://crowdin.com/project/remix-translation/zh-CN

我后面会跟 Remix 团队联系，让他们把所有需要翻译的文档都放到 crowdin 上。

**Comment by drafish** [comment]
目前是 @gongzishuye 自己在审阅，而且只限于 markdown 解析问题。后续计划组织社区力量参与中文内容的审阅。我们需要写一份文档，告诉社区成员去哪里审阅，怎么审阅，等等。
